geoAssembler/tests/test_centre_optimiser.py

```diff
@@ -0,0 +1,68 @@
+import ..optimiser as geoOptimiser
```

Comment:
The test suit is complaining that this isn't valid - you can do `from .. import optimiser`